src/cljs/main/broadfcui/page/notifications.cljs

```diff
@@ -0,0 +1,211 @@
+(ns broadfcui.page.notifications
+  (:require
+   [broadfcui.common :as common]
+   [broadfcui.common.components :as comps]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.style :as style]
+   [broadfcui.nav :as nav]
+   [broadfcui.utils :as utils]
+   [dmohs.react :as r]
+   ))
+
+(def notification-details
+  {"WorkspaceChangedNotification"
+   {:label "Data Added or Changed"
+    :description (str "A Write or Owner can trigger this notification when data is added or changed"
```

Comment:
Wow I can't believe neither myself nor Kate caught the typo. I don't think it should be too specific since it can be triggered whenever a Writer wants to call the API.